app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,32 @@
+/* universal */
+body {
```

Please fix indentation in this file, follow:

```suggestion
body {
  padding-top: 60px;
}

section {
  overflow: auto;
}

...
```